submissions/AndewTu/js-dom/app.css

```diff
@@ -0,0 +1,117 @@
+
+body{
+	max-width: 100%;
+	overflow-x: hidden;
+	margin: 0px;
+	padding: 0px;
+}
+.content{
+	display: flex;
+	justify-content: center;
+}
+.container{
+	max-width: 1280px;
+	height: auto;
+	display: flex;
+	align-items: flex-start;
+	flex-direction: row;
+	flex-wrap: wrap;
+}
+.navi{
+	display: flex;
+	width: 30%;
+	flex-direction: column;
+	text-align: center;
+	padding-top: 10px;
+}
+ul{
+	align-self: center;
+	display: flex;
+	flex-direction: column;
+  	list-style: none;
+  	text-decoration: none;
+  	width: 100%;
+  	padding: 0px;
+}
+li{
+	padding: 10px;
+}
+.btn{	
+	height: 45px;
+	width: 200px;
+	font-size: 1em;
+    font-weight: 600;
+	margin:5px;
+    text-transform: uppercase;
+    border-radius: 50px;
+    cursor: pointer;
+	text-decoration: none;
+	background-color: #fff;
+}
+	.btn#marshall{color: #EE2D25; border: 2px #EE2D25 solid;}
+	.btn#chase{color: #3D4969; border: 2px #3D4969 solid;}
+	.btn#skye{color: #C45376; border: 2px #C45376 solid;}
+	.btn#rubble{color: #D7BE28; border: 2px #D7BE28 solid;}
+	.btn#rocky{color: #6AB950; border: 2px #6AB950 solid;}
+	.btn#zuma{color: #F47736; border: 2px #F47736 solid;}
+	.btn#everest{color: #5AC2B3; border: 2px #5AC2B3 solid;}
+	.btn#marshall:hover, .btn#marshall.active{background-color: #EE2D25;transition: background-color 1s; color: #fff; outline:0 !important;}
+	.btn#chase:hover, .btn#chase.active{background-color: #3D4969;transition: background-color 1s; color: #fff; outline:0 !important;}
+	.btn#skye:hover, .btn#skye.active{background-color: #C45376;transition: background-color 1s; color: #fff; outline:0 !important;}
```

Comment:
1) Don't ever describe selector's rules in a single line.
1) [Selector and Declaration Separation](https://google.github.io/styleguide/htmlcssguide.html#Selector_and_Declaration_Separation)
1) Commonly, using `!important` when you just started project may tell us that you chose a wrong way. If I get right the course of your thoughts - now you may rid of all `outline:0 !important;`, and just one time write:

        .btn:focus {
          outline: none;
        }